src/NServiceBus.Gateway/Settings/GatewaySettings.cs

```diff
@@ -147,7 +147,7 @@ public void TransactionTimeout(TimeSpan timeout)
 #if NETSTANDARD2_0
             return null;
 #endif
-#if NET452
+#if NET472
```

Comment:
that stuff can probably also be removed given we no longer support the XML config approach?